src/main/java/com/hackclub/hccore/playerMessages/UnlinkedSlackMessage.java

```diff
@@ -0,0 +1,21 @@
+package com.hackclub.hccore.playerMessages;
+
+import static net.kyori.adventure.text.minimessage.MiniMessage.miniMessage;
+
+import net.kyori.adventure.text.Component;
+
+public class UnlinkedSlackMessage {
+
+  final static String minimsgSource = """
+      <b><red>Link your MC account with Hack Club Slack</red></b>
+      <gold><b>1.</b></gold> Not on the Slack? Don't worry!
+         Join at <click:open_url:'https://hackclub.com/slack'><hover:show_text:'Open in Browser'>hackclub.com/slack</hover></click>
```

Comment:
I think that the slack link should be underlined to signify that it is a clickable link

Reply:
disagree: minecraft underline formating eats into the next line ingame, and there is a hover message anyway